app/Thread.php

```diff
@@ -332,7 +332,7 @@ public function getBodyWithFormatedLinks(string $body = '') :string
             $body = $this->body;
         }
 
-        $body = \Helper::linkify($this->getCleanBody($body));
+        //$body = \Helper::linkify($this->getCleanBody($body));
```

Comment:
The desired link is displayed in the message after uncommenting this line @smilemittal
Therefore a few other messages are displayed incorrectly and shrinked to half screen size:

![grafik](https://github.com/freescout-helpdesk/freescout/assets/2938748/89d272db-29b3-47fe-b6a0-363d4c4b0409)